clients/admin-ui/src/features/privacy-requests/RequestRow.tsx

```diff
@@ -222,6 +242,7 @@ const RequestRow: React.FC<{
           onBlur={handleBlur}
           shadow="base"
           borderRadius="md"
+          backgroundColor="white"
```

Comment:
This is required because chakra makes disabled buttons have `opacity: 0.4` and the text behind the buttons became visible. The white background fixes it